src/main/java/edu/arizona/kra/proposaldevelopment/bo/ProposalDevelopmentRoutingState.java

```diff
@@ -0,0 +1,316 @@
+        this.sponsorDeadlineTime = sponsorDeadlineTime;
+    }
+
+    public String getPrincipalInvestigatorName() {
+        return principalInvestigatorName;
+    }
+
+    public void setPrincipalInvestigatorName(String principalInvestigatorName) {
+        this.principalInvestigatorName = principalInvestigatorName;
+    }
+
+
+    public String getProposalPersonName() {
+        return proposalPersonName;
+    }
+
+    public void setProposalPersonName(String proposalPersonName) {
+        this.proposalPersonName = proposalPersonName;
+    }
+
+    public String getLeadUnitNumber() {
+        return leadUnitNumber;
+    }
+
+    public void setLeadUnitNumber(String leadUnitNumber) {
+        this.leadUnitNumber = leadUnitNumber;
+    }
+    
+    public String getLeadUnitName() {
+        return leadUnitName;
+    }
+
+    public void setLeadUnitName(String leadUnitName) {
+        this.leadUnitName = leadUnitName;
+    }
+   
+    public String getRouteStopName() {
+        return routeStopName;
+    }
+
+    public void setRouteStopName(String routeStopName) {
+        this.routeStopName = routeStopName;
+    }
+
+    public String getRouteStopCollege() {
+        return routeStopCollege;
+    }
+
+    public void setRouteStopCollege(String routeStopCollege) {
+        this.routeStopCollege = routeStopCollege;
+    }
+
+    public Timestamp getRouteStopDate() {
+        return routeStopDate;
+    }
+
+    public void setRouteStopDate(Timestamp routeStopDate) {
+        this.routeStopDate = routeStopDate;
+    }
+
+    public Boolean isFinalProposalReceived() {
+        return finalProposalReceived;
+    }
+
+    public void setFinalProposalReceived(Boolean finalProposalReceived) {
+        this.finalProposalReceived = finalProposalReceived;
+    }
+
+    public KcPerson getSPSReviewer() {
+        return spsReviewer;
+    }
+
+    public void setSPSReviewer(KcPerson reviewer) {
+        spsReviewer = reviewer;
+        if (reviewer != null){
+            spsReviewerName = reviewer.getFullName();
+            spsPersonId = reviewer.getPersonId();
+        }
+    }
+    
+    public String getSPSReviewerName(){
+        return spsReviewerName;
+    }
+
+    public Boolean isORDExpedited() {
+        return ordExpedited;
+    }
+
+    public void setORDExpedited(Boolean ordExpedited) {
+        this.ordExpedited = ordExpedited;
+    }
+    
+    public String getSPSPersonId() {
+        return spsPersonId;
+    }
+
+    public void setSPSPersonId(String spsPersonId) {
+        this.spsPersonId = spsPersonId;
+    }
+
+    public void setSPSReviewerName(String spsReviewerName) {
+        this.spsReviewerName = spsReviewerName;
+    }
+
+    /**
+     * Returns the route image which can be used to construct the route log link in custom lookup helper code.
+     */
+    public String getRouteLog() {
+        return "<img alt=\"Route Log for Document\" src=\"images/my_route_log.gif\"/>";
```

Comment:
Not sure what HTML is doing here, also this method appears to have no references outside this class.